### ISSUE TITLE
BUG: Spaces created from beta user are not displayed on Admin/Space section

### ISSUE THREAD

**Issue by Comoque1** [cherrytwist/Server#4056]
**Describe the bug**

**To Reproduce**

1. Create a space with beta user
2. Navigate to http://localhost:3000/admin/spaces - spaces created from beta tester are not displayed
3. See error
<img width="1808" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeGc1QlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0470001a4e01b4104cad000ce0ef7328b06cc672/image.png" alt="image.png" />

**Expected behavior**
Spaces created from beta user are displayed on Admin/Space section.

## Additional information
After policy reset is applied, the spaces become visible.

**Comment by bobbykolev** [comment]
Can't reproduce this one. 
STR:
1. create a user;
2. add the user to BETA_TESTERS;
3. Create a free space;

The space is listed in the list of spaces in Global Administration.

_note that I dropped my DB and recreated it_

**More Context**
I saw the issue on Evegeni's machine. He's granted only the PLATFORM_ADMIN privilege for the missing spaces. The client checks for UPDATE, which seems reasonable. Again, working as expected on my end.
